### ISSUE TITLE
Some interesting things to consider

### ISSUE THREAD

**Issue by dw-e8** [bradlnz/Elabor8.Bradl#3]
Hi Brad :)

Things that came up during our interview.

Youtube channels that are MUST_WATCH:

**Nick Chapsas**, Microsoft MVP regularly posting new and cool stuff, best practices and just generally awesome dotnet content
https://www.youtube.com/@nickchapsas

**Dave Farley** - co-author of Continuous Delivery, regularly posting all sorts of great brain-food content: patterns, practices, from TDD to Event-Driven Microservices Architectures ... lots of interviews with thought leaders and heavy hitters too. Great channel.

https://www.youtube.com/c/ContinuousDelivery

Some NUGETS to use _all the time_ ::

FluentAssertions - for making life easier during unit testing / integration testing
Bogus - for generating fake data which makes life easier during unit testing / integration testing

FluentValidations - making life easier to short-circuit bad-parameters and whatnot

**Books that are awesome:**

_Easy to read, good for the mind_
The Pragmatic Programmer
The Phoenix Project
The Unicorn Project

_Deep Tech, career oriented_
Domain Driven Design
Continuous Delivery
The Devops Handbook

Basically you should read these books over the course of the next year! 

Thank you again for all your efforts, time spent and the exceptional quality of your work.

Cheers
DW

**Comment by bradlnz** [comment]
Love it @dw-e8, and thank you so much for detailing what was mentioned in our meeting. I will definitely be buying these books as a read.
